skin.json

```diff
@@ -254,7 +277,8 @@
     "vrIcon": false
   },
   "audio": {
-    "audioLanguage": "none"
+    "audioLanguage": "none",
+    "audioOnly": true
```

Comment:
Should this go somewhere else? I'm up for any suggestions

Reply:
Maybe we should put audioOnly into the 'general' section.